martinLi/assignments/mergeIntervals/lc252/lc252.js

```diff
@@ -0,0 +1,110 @@
+// Problem Statement #
+
+// Given an array of intervals representing ‘N’ appointments,
+//find out if a person can attend all the appointments.
+
```

Comment:
Would recommend including a high level overview / pseudocode of your approach. Also, don't forget to type out the solution twice to help with muscle memory.